src/block/Lever.php

```diff
@@ -91,14 +93,25 @@ public function onNearbyBlockChange() : void{
 	}
 
 	public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player $player = null, array &$returnedItems = []) : bool{
+		$this->toggle();
+
+		return true;
+	}
+
+	public function onProjectileInteraction(Projectile $projectile) : void{
+		if($projectile instanceof WindCharge) {
+			$this->toggle();
+		}
+	}
+
+	public function toggle() : void {
```

Comment:
This should be `private` or `protected`, making it public has implications beyond this PR's scope